src/lib.rs

```diff
@@ -11,6 +11,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+#![cfg_attr(coverage_nightly, feature(no_coverage))]
```

Comment:
- It should have blank line between code and license.
- this `cfg` attribute it looks no effort, please use `cargo clippy` to chheck